.travis.yml

```diff
@@ -41,7 +41,7 @@ env:
         # are in astropy-ci-extras. If your package uses either of these,
         # add the channels to CONDA_CHANNELS along with any other channels
         # you want to use.
-        # - CONDA_CHANNELS='astopy-ci-extras astropy'
+        - CONDA_CHANNELS='astropy-ci-extras astropy'
```

Comment:
I'm not sure we want to remove the comment here, rather than adding the channel info to the LTS builds. (e.g packages that won't test against LTS wont need this, and it would be better for maintainers to change the global settings only when they need it.)